shared/p2p/hobbits/hobbits.go

```diff
@@ -0,0 +1,64 @@
+package hobbits
+
+import (
+	"net"
+	"reflect"
+	"sync"
+
+	peer "github.com/libp2p/go-libp2p-peer"
+	"github.com/prysmaticlabs/prysm/beacon-chain/db"
+	"github.com/prysmaticlabs/prysm/shared/p2p"
+	"github.com/renaynay/go-hobbits/encoding"
+	"github.com/renaynay/go-hobbits/tcp"
+	log "github.com/sirupsen/logrus"
+	ttl "github.com/ReneKroon/ttlcache"
+)
+
+type HobbitsNode struct {
+	sync.Mutex
+	NodeId       string
+	Host         string
+	Port         int
+	StaticPeers  []string
+	PeerConns    map[peer.ID]net.Conn
+	feeds        map[reflect.Type]p2p.Feed
+	Server       *tcp.Server
+	DB           *db.BeaconDB
```

Comment:
The hobbits node should not use a database.

Reply:
+1